examples/tinywl/main.cpp

```diff
@@ -55,7 +55,12 @@ class TinywlServer : public QObject
 
 private Q_SLOTS:
     void onNewOutput(QWOutput *output);
+#if WLR_VERSION_MINOR < 18
     void onNewXdgSurface(wlr_xdg_surface *surface);
+#endif
+    void onNewXdgToplevel(wlr_xdg_toplevel *surface);
```

Comment:
`#else`?

Reply:
Before 0.18, entry is onNewXdgSurface, it checks role type and call onNewXdgToplevel or popup.
In 0.18, entry is onNewXdgPopup, and it needs to make a initial commit configure